SPEC.md

```diff
@@ -279,7 +279,15 @@ The fuzz tests aim to buttress the message conformance tests with extra verifica
 
 ### ZG-RESISTANCE-003
 
-    Reserved
+    The node rejects various random bytes post-handshake.
+
+    -> random bytes
```

Comment:
We could expand these cases to include some flipped bits on otherwise valid messages, random portions of valid messages being random etc...

Reply:
> We could expand these cases to include some flipped bits on otherwise valid messages, random portions of valid messages being random etc...

I've been doing that unofficially with most messages - and on top of that, emulating attacks with a specific focus on certain message fields - that will be documented separately.

I'll see if some of that work can be converted into official tests here, but the results (for the assertions) are the same, basically
It's tricky with this protocol since most messages do not offer any response - and the 'test results' are observed through the node's logs.